src/coin/mocks/ReadableTxn.go

```diff
@@ -0,0 +1,36 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
This is an internal interface . No mocks needed .

Reply:
Usage instances should be replaced with concrete type implementing the internal interface.

Reply:
> Usage instances should be replaced with concrete type implementing the internal interface.

Forget about this previous comment .

The only thing needed is to move this class somewhere under `src/coin/skycoin/` sub package tree (e.g. `src/coin/skycoin/skymocks`) . Code under `src/coin/mocks` should correspond to types defined in `src/core` .